app/src/main/java/com/example/bigowlapp/utils/NotificationListenerManager.java

```diff
@@ -37,6 +45,10 @@ public void listen(Context context) {
                         if (notification.getType() == Notification.Type.AUTH_BY_PHONE_NUMBER_FAILURE) {
                             handleAuthByPhoneNumberFailure((AuthByPhoneNumberFailure) notification);
                         }
+
+                        if(notification.getType() == Notification.Type.SMS_INVITATION_REQUEST){
+                            handleSmsInvitationRequest((SmsInvitationRequest) notification, context);
+                        }
```

Comment:
I think you should just make this an if else statement and  make handleSmsInvitationRequest into something that takes all notifs.

This would allow all notifs to have a message, and its a pretty simple change. Right now only SMS_INVITATION_REQUEST is sending notifs